src/components/SettingsModal.js

```diff
@@ -46,9 +59,14 @@ export default function SettingsModal({ setDisplayModal }) {
             max="30"
             value={sliderValue}
             step="5"
-            onChange={(e) => handleSliderChange(e)}
+            onChange={handleSliderChange}
           />
-          <div className="slider-value">{sliderValue}</div>
+          <div
+            aria-label="current fetch frequency selection by number of seconds"
```

Comment:
I _think_ that's a big improvement! The thing to do would be to navigate to that element using VoiceOver or NVDA and move the slider around. Does it announce the changes?

Reply:
It appears so. 🤞  That was my 1st time ever maneuvering with VoiceOver though, so another pair of eyes (ears) on it could be useful.